packages/flutter_tools/lib/src/ios/devices.dart

```diff
@@ -762,10 +756,12 @@ class IOSDevicePortForwarder extends DevicePortForwarder {
     @required String id,
     @required IProxy iproxy,
     @required OperatingSystemUtils operatingSystemUtils,
+    @required IOSDeviceInterface interfaceType,
```

Comment:
ultra nit / feel free to ignore: IOSDeviceConnectionInterface? Just interface is a bit vague.

Reply:
Done.